website-next/src/shared/components/fonts/style.css

```diff
@@ -7,8 +7,6 @@
  * (https://github.com/css-modules/postcss-modules-values/pull/23)
  */
 
-@import url("//hello.myfonts.net/count/322627");
```

Comment:
Don't we need to keep this so they can keep track of impressions? We could remove local versions of the fonts I suppose. Or am I missing something?